WTWSMS/events/religious_events.txt

```diff
@@ -853,6 +871,10 @@ character_event = {
 				factor = 0.25
 				tier = EMPEROR
 			}
+			modifier = {
+				factor = 1.20
+				trait = arbitrary
+			}
```

Comment:
Arbitrary characters act randomly, with no care for law, justice, or anything but their own feelings and mood that day.

You are right, though, that is too high. I will lower it.